system/ubox: add priority filter for syslog messages, which are collecting by logd

SuprMaks opened the pull request:
I noticed that syslog is not reacting on "Log output level" parameter in luci/admin/system menu.
If I select "emergency" for example, syslog still keep catching messages with "notice" priority.

I found topic with the same problem: [link](https://forum.openwrt.org/t/log-output-level-in-syslog-is-not-working/34656)

So I did a simple patch for myself and maybe it will be useful for you too.

adrianschmutzler commented:
Hi, your Pull Request has formal issues. Please have another look at https://openwrt.org/submitting-patches

SuprMaks committed:
system/ubox: add priority filter for syslog messages, which are collecting by logd

Syslog is not reacting on "Log output level" parameter in luci/admin/system menu

Signed-off-by: Maksim Kapustin <8supermaks8@gmail.com>

8SuperMaks8 commented:
One commit, and I rewrote a commit msg.

adrianschmutzler commented:
The prefix for your commit title would be just "ubox:". To find out about this, an easy way is to just look at how previous patches to this packages have been named.
Despite, please shorten the title so it is not wrapped into the message.

Concerning the patch you add to the "patches" folder inside your commit, please submit that patch against the ubox subproject directly: https://openwrt.org/docs/techref/ubox
To do that, make it a proper separate patch and submit it to the openwrt-devel mailing list with "ubox" in the patch prefix: "[PATCH ubox]".

adrianschmutzler commented:
Have you submitted the ubox patch already?

adrianschmutzler commented:
This PR seems to be abandoned, so I'm closing it now. If there is interest in continuing this subject, please send a patch against the ubox repository as explained above first.
